src/Features/CSharp/Portable/ImplementInterface/CSharpImplementExplicitlyCodeRefactoringProvider.cs

```diff
@@ -49,11 +48,18 @@ protected override bool CheckMemberCanBeConverted(ISymbol member)
         {
             var solution = project.Solution;
 
+            // We don't need to cascade in this search, we're only explicitly looking for direct
+            // calls to our instance member (and not anyone else already calling through the
+            // interface already).
+            //
+            // This can save a lot of extra time spent finding callers, especially for methods with
+            // high fan-out (like IDisposable.Dispose()).
+            var findRefsOptions = FindReferencesSearchOptions.Default.WithCascade(false);
```

Comment:
Planning to use this?

Reply:
i think it should work by globally changing state and havin gthat affect everyone. :)